docs/releases/2.1.txt

```diff
@@ -178,6 +178,8 @@ Models
   :class:`~django.db.models.DateField` and
   :class:`~django.db.models.DateTimeField` to the Monday of a week.
 
+* Expressions can now be made negative using the unary negative operator.
```

Comment:
I would say "a minus sign" instead of "the unary negative operator".

Reply:
OK. For the behaviour, I have added the word ```negation``` in docs/ref/models/expressions.txt. Squashed this commit into the last one to keep a single commit.